packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -61,10 +77,13 @@ class CpuProfilerController
   ValueListenable<String> get userTagFilter => _userTagFilter;
   final _userTagFilter = ValueNotifier<String>(userTagNone);
 
-  @visibleForTesting
-  final dataByTag = <String, CpuProfileData>{};
+  Iterable<String> get userTags =>
+      cpuProfileStore.lookupProfile(label: userTagNone)?.userTags ?? [];
```

Comment:
`[]` ==> `const []`
to avoid an allocation for the common case.

Reply:
done